src/components/lib/VRadio/VRadio.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <div class="flex flex-center justify-space-between">
+    <label class="body-1">{{ label }}</label>
```

Comment:
Can we add a `for` attribute to this label?

Reply:
done. [commit link](https://github.com/arcana-network/developer-dashboard/pull/50/commits/0c4a00809437daf0c6e56487c0b5a32056014fe5)